SR2024/2023-09-23-place-confirmation.md

```diff
@@ -0,0 +1,51 @@
+---
+to: Student Robotics 2024
+subject: Confirming your place at Student Robotics 2024
+---
+
+You're in! We're happy to confirm you've got a place at Student Robotics 2024!
+
+As the new school year starts, so does Student Robotics. The year starts with a
+"Kickstart" event which gives competitors and team supervisors an introduction
+to this year's competition and overview of how to build and engineer their
+robot.
+
+At various points in the year we will have a number of "Tech Days" at locations
+around the UK. These provide an opportunity for your team to get hands-on help
+from our volunteer mentors, make progress on their robot and meet other teams.
+
+Throughout the year your team will be able to get support for our mentors via
+[Discord](https://studentrobotics.org/docs/tutorials/discord). In addition to
+shared channels where any team can ask questions, each team will have their own
+private text chat channel and team supervisors can request voice & video
+channels too. These offer a space for teams to discuss their robot and get
+direct support from our mentors at any time. We will send out invites to Discord
+separately.
+
+This year **Kickstart is fully virtual**. We strongly encourage teams to meet up
```

Comment:
Issue: We should link to the event here https://studentrobotics.org/events/sr2024/virtual-kickstart/

Reply:
df72bf26ec7d1bcb9527e1c007032662edd6040d & 5aecf0a5240ec54eab10fac86b15c7f0537c4847